lib/src/client_messages.dart

```diff
@@ -199,35 +246,54 @@ class _BindMessage extends _ClientMessage {
   }
 
   int applyToBuffer(ByteData buffer, int offset) {
-    buffer.setUint8(offset, _ClientMessage.BindIdentifier); offset += 1;
-    buffer.setUint32(offset, length - 1); offset += 4;
+    buffer.setUint8(offset, _ClientMessage.BindIdentifier);
+    offset += 1;
+    buffer.setUint32(offset, length - 1);
+    offset += 4;
 
-    offset = applyStringToBuffer("", buffer, offset); // Name of portal - currently unnamed portal.
-    offset = applyStringToBuffer(statementName, buffer, offset); // Name of prepared statement.
+    offset = applyBytesToBuffer(
+        [0], buffer, offset); // Name of portal - currently unnamed portal.
```

Comment:
its pedantic, but can we move or shorten these trailing comments to avoid having dartfmt split these trivial lines.. there's just a few of them